v0/destinations/active_campaign/transform.js

```diff
@@ -608,11 +572,8 @@ const trackRequestHandler = async (message, category, destination) => {
   payload.actid = destination.Config.actid;
   payload.key = destination.Config.eventKey;
   payload.visit = encodeURIComponent(
-    `{email : ${
-      message.context.traits.email
-        ? message.context.traits.email
-        : message.context.traits.traits.email
-    }}`
+    `{"email":"${get(message, "context.traits.email") ||
+      get(message, "context.traits.traits.email")}"}`
```

Comment:
[Non-blocking] Why are we even supporting `traits.traits` here and in other places?

Reply:
We were supporting it earlier so it has been kept intact for backward compatibility.

Reply:
@yashasvibajpai can take up this cleanup item as he is working on refactoring this destination along with other CFD for Active Campaigns